holidays/countries/austria.py

```diff
@@ -21,16 +21,32 @@ class Austria(HolidayBase, ChristianHolidays, InternationalHolidays):
     default_language = "de"
     supported_categories = (BANK, PUBLIC)
     supported_languages = ("de", "en_US", "uk")
-    subdivisions = ("1", "2", "3", "4", "5", "6", "7", "8", "9")
+    subdivisions = (
+        "1",  # Burgenland.
+        "2",  # Kärnten.
+        "3",  # Niederösterreich.
+        "4",  # Oberösterreich.
+        "5",  # Salzburg.
+        "6",  # Steiermark.
+        "7",  # Tirol.
+        "8",  # Vorarlberg.
+        "9",  # Wien.
+    )
+    subdivisions_aliases = {
+        "B": "1",
+        "K": "2",
+        "N": "3",
+        "O": "4",
+        "S": "5",
+        "St": "6",
+        "T": "7",
+        "V": "8",
+        "W": "9",
```

Comment:
```suggestion
        "Burgenland": "1",
        "Kärnten": "2",
        "Niederösterreich": "3",
        "Oberösterreich": "4",
        "Salzburg": "5",
        "Steiermark": "6",
        "Tirol": "7",
        "Vorarlberg": "8",
        "Wien": "9",
```

This is how imagine aliases resolving the initial AT irrelevant subdivision names problem.
@sphh what do you think would be a reasonable addition to this mapping?